.github/workflows/merge.yml

```diff
@@ -0,0 +1,81 @@
+name: merge
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+on:
+  push:
+    branches:
+      - 'v2'
+    tags:
+      - 'v*'
+  workflow_dispatch:
+
+jobs:
+  e2e:
+    name: Build and test
+    runs-on: ${{ matrix.os }}
+    timeout-minutes: 15
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [desktop-windows, desktop-macos, desktop-m1]
+        # mode: [plugin, standalone]
+        mode: [plugin]
+    env:
+      GO111MODULE: "on"
+    steps:
+      - name: test-action-unsafe-checkout
+        if: ${{ contains(matrix.os, 'desktop-windows') }}
+        run: |
+          git config --global --add safe.directory C:/actions-runner/_work/compose/compose
+      - uses: actions/checkout@v3
+
+      - uses: actions/setup-go@v3
+        with:
+          go-version-file: go.mod
+          cache: true
+          check-latest: true
+
+      - name: Check arch of go binary
+        run: |
+          which go
+          file $(which go)
+        if: ${{ !contains(matrix.os, 'desktop-windows') }}
```

Comment:
I think it's safe to get rid of this -- it was a holdover from when we had to do Go M1 installs manually before it was available from the action I think.

Reply:
Sounds good!